process/dataValidators/nilTxHandlerProcessValidator.go

```diff
@@ -0,0 +1,17 @@
+package dataValidators
+
+import "github.com/ElrondNetwork/elrond-go/data"
+
+// nilTxHandlerProcessValidator represents a tx handler validator that doesn't check the validity of provided txHandler
+type nilTxHandlerProcessValidator struct {
```

Comment:
name too long: nilTxValidator should be enough

Reply:
Done